component/storage.md

```diff
@@ -312,6 +312,12 @@ return [
             // prefix.
             //
             'prefix' => '',
+
+            //
+            // Any additional options you want to pass to S3 client that are not
+            // listed above (e.g. 'use_path_style_endpoint').
```

Comment:
I think it's worth mentioning that the option is available since `v2.9`, because we do not have versioning of the documentation at the moment =(